File: examples/Person.java
class Person{
    // Propiedades Privadas
    private String name
    private int age
    // Metodos Publicos
    // constructor
    public Person(String name, int age){
        this.name = name;
        this.age = age;
    }
    // Getters
    public String getName(){
        return this.name;
    }
    public int getAge(){
        return this.age;
    }
    // Setters
    public void setName(name){
        this.name = name;
    }
    public void setAge(age){
        this.age = age;
    }
}
// Instancia
Person user = new Person("cristian",18)
user.setAge(33)

File: examples/Person.js
class Person{
    // Propiedades Privadas
    #name
    #age
    // Metodos Publicos
    // constructor
    constructor(name, age){
        this.#name = name;
        this.#age = age;
    }
    // Getters
    getName(){
        return this.#name;
    }
    getAge(){
        return this.#age;
    }
    // Setters
    setName(name){
        this.#name = name;
    }
    setAge(age){
        this.#age = age;
    }
}
// Instancia
const user = new Person("cristian",18)
user.setAge(33)